simplq/src/components/pages/Admin/AddMember.jsx

```diff
@@ -1,29 +1,19 @@
 import React from 'react';
-import { Collapse, ListItemIcon } from '@material-ui/core';
 import AddIcon from '@material-ui/icons/Add';
-import { ExpandLess, ExpandMore } from '@material-ui/icons';
-import ListItem from '@material-ui/core/ListItem';
-import ListItemText from '@material-ui/core/ListItemText';
-import JoinQueueForm from '../Join/Form';
+// import JoinQueueForm from '../Join/Form';
```

Comment:
remove... ideally the codebase should not have any dead code / comment.

Reply:
I kept this to use for the form, but you're right. I'll remove for now.